include/boost/url/grammar/unsigned_decimal.hpp

```diff
@@ -50,6 +49,12 @@ struct unsigned_decimal
     }
 
 private:
+    BOOST_STATIC_ASSERT(
+        std::numeric_limits<
+            Unsigned>::is_integer &&
+        ! std::numeric_limits<
+            Unsigned>::is_signed);
+
     static
     void
     parse(
```

Comment:
this function definition should go in `<boost/url/grammar/impl/unsigned_decimal.hpp>`